src/lib/kdb/kdb5.c

```diff
@@ -323,6 +323,8 @@ copy_vtable(const kdb_vftabl *in, kdb_vftabl *out)
     out->refresh_config = in->refresh_config;
     out->check_allowed_to_delegate = in->check_allowed_to_delegate;
     out->free_principal_e_data = in->free_principal_e_data;
+    /* TODO: should we first check minor version? */
```

Comment:
Yes, if I understand the question?

Reply:
I'll add it the same check as free_principal_e_data had initially.